graphql/schema/types/scene.graphql

```diff
@@ -39,6 +39,9 @@ type Scene {
   path: String!
   phash: String
   interactive: Boolean!
+  created_at: Time!
+  updated_at: Time!
+  file_mod_time: Time!
```

Comment:
Similarly, this throws an error if the scene `file_mod_time` is null, which can happen.